app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
+	page_title.empty? ? base_title : "#{page_title} | #{base_title}"
```

Comment:
1. plz, remove code ` = ""`
2. Use I18n for `"Ruby on Rails Tutorial Sample App"` text
3. you should use `blank?`, don't use `empty?`